modules/understanding-upgrade-channels.adoc

```diff
@@ -7,79 +7,89 @@
-minor release (4.5.z) when the next minor release is available.
+The fast channel, containing only GA releases, is updated with new
+{product-version} patch versions as soon as Red Hat declares they are generally
+available. Use this channel if you wish to receive updates as soon as they are
+available or for your pre-production environments when participating in the connected
+customer program. This channel will contain all z-stream ({product-version}.z)
+updates but will not suggest upgrades to the next minor release (4.5.z) when the next
+minor release is available. This channel also includes the upgrade from the previous
+4.3 release.
 
 = stable-{product-version}
 
-The stable channel will contain updates on a time delay as they are gradually
-rolled out to customers based on data from our SRE teams, support services, and
-pre-production and production environments that participate in our connected
-customer program, rather than being immediately available as they are in the fast
-channel. For patch and CVE fixes this can range from several hours to a day and
-allows an extra period of assessment in how the software performs. If issues are
-detected during rollout, upgrades to that version may be blocked in both the fast
-and stable channels, and a new version may be introduced that will be the new
-preferred upgrade target.
+The stable channel, containing only GA releases, will contain updates on a time delay
+as they are gradually rolled out to customers based on data from our SRE teams,
+support services, and pre-production and production environments that participate in
+our connected customer program, rather than being immediately available as they are
+in the fast channel. For patch and CVE fixes this can range from several hours to a
+day and allows an extra period of assessment in how the software performs. This
+channel also includes the upgrade from the previous 4.3 release.
+
+= Fast and Stable Channel Usage and Strategies
+
+The fast-{product-version} and stable-{product-version} channels present a choice between receiving
+GA updates as soon as they are available or allowing Red Hat to control the rollout
+of those updates. If issues are detected during rollout, upgrades to that version may
+be blocked in both the fast and stable channels, and a new version may be introduced
+that will be the new preferred upgrade target.
 
 Customers can improve this process by configuring pre-production systems on the
 fast channel, production systems on the stable channel, and participating in Red
-Hat’s connected customer program - this allows Red Hat to observe the impact of
-updates on your specific hardware and software configurations. Future releases may
+Hat’s connected customer program. This program allows Red Hat to observe the impact
+of updates on your specific hardware and software configurations. Future releases may
 improve or alter the pace at which updates move from the fast to the stable channels.
 
```

Comment:
Rewording ideas for the *Switching Between Channels* section below:

> It is supported for customers to switch from the stable-{product-version} channel to fast-{product-versoin} at any time.  Customers may also switch to the candidate-{product-version} at any time without waiving support, but subsequent updates in the candidate-{product-version} channel may waive support (e.g. if that update was not also in the fast-{product-version} channel when it was initiated).  Customers may be able to switch from candidate-{product-version} to fast-{product-version}, depending on whether their current release is supported or not.  Customers will always be able to switch from fast-{product-version} to stable-{product-version}, although there may be a delay of up to a day if their release was recently promoted to fast-{product-stable} while they wait for the release to be promoted to stable-{product-version} for their cluster.  If you change to a channel that does not include your current release, an alert will fire, no updates will be recommended, and you can safely change back to your original channel at any point.

Reply:
Using `may be` for certain situations is a bad idea IMO. We should be very clear about what is possible and which is not.  For example `Customers may be able to switch from candidate-{product-version} to fast-{product-version}, depending on whether their current release is supported or not.` is not something we want to communicate as part of the document text. May be we can add a section which talks about these corner case exceptions.

Reply:
> May be we can add a section which talks about these corner case exceptions.

I think the *Switching Between Channels* section is that corner-case section.